docs/source/docker/getting_started/first_steps.rst

```diff
@@ -248,16 +255,6 @@ After the job was processed, the selected data will be accessible in the configu
 report can be accessed from the compute worker runs page mentioned just above.
 
 
-There's an alternative stopping condition to `n_samples`, the `min_distance`
-
-- :code:`stopping_condition.min_distance` **0.2** would remove all samples which are
```

Comment:
is this no longer true? Maybe move it to where we use it in one of the examples

Reply:
We now have an example for it when explaining the DIVERSIFY strategy:

```json
"strategy": {
    "type": "DIVERSIFY",
    "stopping_condition_minimum_distance": 0.2
}
```

Reply:
I was more referring to the removal of the information as it was not moved to a new location

Reply:
In my eyes, the concept of the `stopping_condition_minimum_distance` is explained well enough in the diversity section. Thus I see this as solved.
How would you  explain/show it @japrescott